### DISCUSSION TITLE
鹤形拆字的不足之处

### DISCUSSION THREAD

**Discussion by gaboolic** [gaboolic/rime-config#32] [General]
文是wx亠　乂，方却是dw 丶万

家是bu，蒙却是cn 拆    分：艹　冖　二　丿　乛　丿　丿　丿　ㄟ

**Comment by RubberTale** [comment]
我觉得很好理解啊。
文无法拆成丶和下面这个字根，只能拆成亠和乂
而方也是，无法拆成亠和下面这个字根，所以拆成丶和万。
